### PR TITLE
[v11] [Discover] Add ons for database flow

### PR DESCRIPTION
**Pull request by kimlisa**
Backport #19063 to branch/v11

### CODE REVIEW

**Review by rosstimothy — APPROVED** [review]

**Review by marcoandredinis — APPROVED** [review]

File: lib/web/ui/server.go [hunk 1]
```diff
@@ -368,3 +371,27 @@ func MakeDesktopServices(windowsDesktopServices []types.WindowsDesktopService) [
 
 	return desktopServices
 }
+
+// stripProtocolAndPort returns only the hostname of the uri.
+// Handles uri's with no protocol eg: for some database connection
+// endpoint the uri can be in the format "hostname:port".
+func stripProtocolAndPort(uri string) string {
```
Comment (zmb3) [inline lib/web/ui/server.go:378]: I realize this is a backport and is already approved, but out of curiousity why are we not using `url.Parse` here?

Comment (marcoandredinis) [reply lib/web/ui/server.go:378]: :sweat_smile: https://github.com/gravitational/teleport/pull/19063#discussion_r1041211774

Comment (zmb3) [reply lib/web/ui/server.go:378]: Hah! Thanks Marco! 😄

Comment (kimlisa) [reply lib/web/ui/server.go:378]: sorry for my subpar grammar :sweat_smile:, fixed the grammer here and sneaked it in here for master: https://github.com/gravitational/teleport/pull/19285/commits/a9be35ab61fb56e1ef0e6aa3976280958e5555b9

**Pull request merged**